runtime/compiler/runtime/IProfiler.hpp

```diff
@@ -397,7 +397,12 @@ class TR_IPBCDataCallGraph : public TR_IPBytecodeHashTableEntry
    void * operator new (size_t size) throw();
    void * operator new (size_t size, void * placement) {return placement;}
 
+#ifdef OMR_GC_COMPRESSED_POINTERS
+   // Set the higher 32 bits to zero under compressedref to avoid assertion in debug build
+   static const uintptrj_t IPROFILING_INVALID = 0x00000000FFFFFFFF;
```

Comment:
Which assertion exactly?

Reply:
It's assertion in `CallSiteProfileInfo::setClazz`, called by `setInvalid` with `IPROFILING_INVALID`

Reply:
Sounds like a reasonable thing to document in the comment if someone wants to revisit this in the future so they know the exact assertion which fires.

Reply:
Good point. I modified the comment to point out the assertion we're trying to avoid.